### PR TITLE
Added Bank Removing

### PR DESCRIPTION
**Pull request by loudnessofthelambs**
Added ability to delete a bank account for the AbstractPlayerAccount class

### CODE REVIEW

File: src/main/java/org/kaiaccount/account/inter/type/player/AbstractPlayerAccount.java [hunk 1]
```diff
@@ -135,6 +136,19 @@ public CompletableFuture<TransactionResult> multipleTransaction(
         return account;
     }
 
+	public boolean deleteBankAccount(@NotNull String name) {
```
Comment (mosemister) [inline src/main/java/org/kaiaccount/account/inter/type/player/AbstractPlayerAccount.java]: Could we have a interface method for this please?
https://github.com/EconToolBox/AccountInterface/blob/main/src/main/java/org/kaiaccount/account/inter/type/named/bank/BankAccount.java

File: src/main/java/org/kaiaccount/account/inter/type/player/AbstractPlayerAccount.java [hunk 2]
```diff
@@ -135,6 +136,19 @@ public CompletableFuture<TransactionResult> multipleTransaction(
...
+		Optional<PlayerBankAccount> accountOptional = this.getBank(name);
+		if (accountOptional.isEmpty()) {
+			return false;
```
Comment (mosemister) [inline src/main/java/org/kaiaccount/account/inter/type/player/AbstractPlayerAccount.java]: The ``tab`` space seems a bit too large. If your using intellijj then you should be able to click the format button to fix this. 

I personally have ``Format, optimise imports and save`` all on the save key. You can do that with Macros. The example in the Macro setup guide is ``Format and save`` so should give you a good idea if you choose to use it

https://www.jetbrains.com/help/idea/using-macros-in-the-editor.html#reformat_on_save

**Review by mosemister — CHANGES_REQUESTED** [review]
Overall good start, just 2 minor changes

File: src/main/java/org/kaiaccount/account/inter/type/player/AbstractPlayerAccount.java [hunk 3]
```diff
@@ -135,6 +135,19 @@ public CompletableFuture<TransactionResult> multipleTransaction(
         return account;
     }
 
+    public @NotNull boolean deleteBankAccount(@NotNull String name) {
```
Comment (mosemister) [inline src/main/java/org/kaiaccount/account/inter/type/player/AbstractPlayerAccount.java]: Mind just annotating this with a ``@Override`` to just make it clear its coming from a method outside the class and that the class is modifying the behaviour

**Review by mosemister — CHANGES_REQUESTED** [review]
Just a extremely minor change

**Review by mosemister — DISMISSED** [review]
Approved

**Comment by loudnessofthelambs** [comment]
I rewrote this to work with a PlayerBankAccount instead of a name, can this be reapproved

File: src/main/java/org/kaiaccount/account/inter/type/player/PlayerAccount.java [hunk 4]
```diff
@@ -27,6 +28,9 @@ public interface PlayerAccount<Self extends PlayerAccount<Self>> extends Account
     @NotNull
     PlayerBankAccount createBankAccount(@NotNull String name);
 
+	@NotNull
+	boolean deleteBankAccount(@NotNull PlayerBankAccount account);
```
Comment (mosemister) [inline src/main/java/org/kaiaccount/account/inter/type/player/PlayerAccount.java:32]: Extra indent -> recommend using Intellij auto format

**Review by mosemister — CHANGES_REQUESTED** [review]

**Comment by mosemister** [comment]
Due to the lack of process. I am taking over the task, I have copied the branch with #6

**Pull request closed without merging**